rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.78"
+channel = "1.79"
```

Comment:
I think we've been neglecting this the last couple of bumps, but we should update `Tiltfile` too.

Reply:
Done